kuma/landing/templates/landing/homepage.html

```diff
@@ -82,7 +82,7 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
         </ul>
       </div>
       <div class="column-home-features">
-        <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Zones') }}</span></h3>
+        <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Mozilla<br> Docs') }}</span></h3>
```

Comment:
I just copied how the other multi-line headings on the home page are coded (Developer Tools and Web Platform). Not sure the difference between the two techniques but happy to change if what you are suggesting is superior.

[The docs](https://kuma.readthedocs.org/en/latest/localization.html#making-strings-localizable) suggest `{{ _('Mozilla<br>Docs')|safe }}` is the correct format.